content/1.7/back-office/quick-access/add-edit-quick-access.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Add / edit quick access
+weight: 1
+---
+
+# Add / edit quick access
+
+The following fields/toggles are displayed:
+
+- **Name**: this field is mandatory. When trying to save and if the field is empty, the following error message is displayed: "The field name is required at least in _default language_." The following tooltip is displayed: "Forbidden characters:<>;=#{}". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down. 
+ - **URL**: this field is mandatory. When trying to save and if the field is empty, the following error message is displayed: "The link field is required.".
+The following tooltip is displayed: "If it's an URL that comes from your back office, you MUST remove the security token.".
+
```

Comment:
I've renamed the tooltip to the limit count of the characters in the input instead of the token information, not sure if that's okay?

Reply:
The existing tooltip should stayed mentionned and we can furthermore add the number of maximum characters allowed in it

Reply:
@marionf I agree with you regarding the tooltip. What do you think if an error message with the The maximum allowed characters will be displayed when the limit is exceeded to avoid the risque of loosing some characters without any notification? 
@Saimis777  The first point is still missing : an incorrect URL redirect to the dashboard page. @marionf  What do you think?

Reply:
This should be discussed with the product content manager and designer to standardize the behavior on all back-office pages
For now, I suggest just mentioning the limit of characters in the spec.
I don't understand what you are talking about with the URL, could you give more information?

Reply:
@marionf When entering an incorrect URL the direction will be the dashboard page as you can see in the following screen recording 
https://watch.screencastify.com/v/judV19q9Egh98DnvVwcs

Reply:
@marwachelly so what should be the course of action regarding the URL issue? Should it be written in the specs that only appropriate URL would redirect correctly? And random numbers or letters will just redirect to the dashboard?

Reply:
@marionf  what do you think?

Reply:
Thank you @marwachelly for the video, I better understand what you mean now 👍 
Obviously, the behavior here isn't appropriate and a user shouldn't be able to add an invalid link 😅 
In this case, the best thing to do is [to create an issue](https://github.com/PrestaShop/PrestaShop/issues/new/choose) to describe this inappropriate behavior and then, link the issue in the spec

Reply:
@marionf @marwachelly but is it actually a bug? For example for me it does not even redirect to the dashboard, I just receive an error that the URL does not exist when trying to use it.

I think this might be left like this for a purpose, because there are many users/merchants that are using for example local host, or random servers locally, which have various different links that sometimes can be even extremely weird, so it's hard to define what is a correct URL. 

As an example my URL that I test on is -> 9b97a20862b9.eu.ngrok.io

Reply:
@marwachelly  Also, redirection to the dashboard can also mean cache, clearing your cache might just solve the issue, and not redirect to the dashboard, but rather browser showing incorrect URL input.

Reply:
Thank you for your feedback @Saimis777, it's interesting 👍 You are right, this kind of URL (9b97a20862b9.eu.ngrok.io) is valid and should be accepted.
However, the URL given by @marwachelly as an example in the video, shouldn't be accepted because there is no extension and it's not an URL.

Here, the goal isn't to describe right now what should be accepted or not and what should happen when an URL considered invalid is filled. The goal is just to create an issue to say that the current behavior is inappropriate in some cases (the one given by Marwa). The wanted behavior and what should be accepted or not in the URL field will be defined later.

Reply:
@marionf thanks for the explanation, so I will raise this bug, and link it in the specs.

Reply:
@marionf @Saimis777 Thanks for the feedback and the explanation. PR is QA approved.